samples/.env.test

```
@@ -46,6 +46,15 @@ MONGODB_VERIFY_ON_START=false
 SLACK_BOT_TOKEN=xoxb-test-token
 SLACK_SIGNING_SECRET=test-signing-secret
 
+# --- GitHub webhook (github-webhook) ---
+# Unlike the keys above, this one IS used: the suite verifies a real HMAC against
+# it, with the signature committed alongside the payload it covers. That works
+# because GitHub's scheme has no timestamp, so a committed signature never goes
+# stale — which is what lets the sample prove the accept path and not just the
+# reject one.
+GITHUB_WEBHOOK_SECRET=test-webhook-secret
+GITHUB_WEBHOOK_PATH=hook
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_WEBHOOK_PATH key should go before the GITHUB_WEBHOOK_SECRET key

(UnorderedKey)